core/version.go

```diff
@@ -9,6 +9,7 @@ import (
 var (
 	Ver0_13_2 = semver.MustParse("0.13.2")
 	Ver0_13_4 = semver.MustParse("0.13.4")
+	Ver0_14_0 = semver.MustParse("0.14.0")
```

Comment:
@infrmtcs I saw something similar in one of your PRs signaling it here when the time comes of merging main with 0.15.0